content/general/pubsub.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Pub/Sub
+section: general
+index: 23
+---
+
+Pub/Sub is shorthand for the Publish/Subscribe architectural pattern, which is a popular pattern used for realtime data delivery. This messaging pattern lets any number of publishers publish data, ideally to a data channel/topic, which could be subscribed to by any number of subscribers. The important thing to note about Pub/Sub is that publishers and subscribers are completely decoupled. In addition, once subscribed, the subscribers no longer have to poll the server or data provider to check if there’s any new data that they need to be aware of; instead, they will be notified of it, as it becomes available.
+
+<a href="/images/diagrams/pub-sub.png" target="_blank">
+  <img src="/images/diagrams/pub-sub.png" style="width: 100%" alt="Pub/Sub in Ably">
+</a>
+
+For instance, consider a location tracking application. The subscriber will continue to receive updates in real time for as long as the connection remains connected, and the client is subscribed for updates.
+
+<a href="/images/diagrams/ably-pubsub.png" target="_blank">
+  <img src="/images/diagrams/ably-pubsub.png" style="width: 100%" alt="Pub/Sub in Ably">
+</a>
+
+As seen in the illustration above, you can implement Pub/Sub easily by using Ably as the intermediary realtime messaging platform. Your publishers and subscribers can attach to named "channels":/realtime/channels-messages and Ably is responsible for routing the right message to the right client in real time (typically within 60ms globally).
```

Comment:
With the "(typically within 60ms globally)", it could be good to hyperlink to our metrics (https://status.ably.io/status)